test/gleam/list_test.gleam

```diff
@@ -4,11 +4,13 @@ import gleam/list
 import gleam/should
 
 if erlang {
-  const recursion_test_cycles = 999999
+  // const recursion_test_cycles = 2
```

Comment:
```suggestion
```

Reply:
I'll remove this.

That was there to make it easy to disable the slower tests.

Also for flatten it runs less cycles but only if the number is `> 2`

```gleam
  case recursion_test_cycles > 2 {
    True ->
      list.repeat([[1]], recursion_test_cycles / 50)
      |> list.flatten()
    False -> []
  }
```